src/components/twoFactorAuth/twoFactorSetupDialog.styles.scss

```
@@ -317,7 +317,9 @@
 	&__successIcon {
 		width: 120px !important;
 		height: 120px !important;
-		color: var(--m3-success, #0a882f);
+		// M3 brand role, not a traffic-light green: ORISO signals success with
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)